### PR TITLE
Update DeviceInfo.kt

### PR DESCRIPTION
**Pull request by hcors**
Updated to add "bnrv510" as @pazos asked

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/android-luajit-launcher/204)

### CODE REVIEW

**Comment by hcors** [comment]
https://github.com/koreader/koreader/issues/4551

File: app/src/org/koreader/launcher/DeviceInfo.kt [hunk 1]
```diff
@@ -139,7 +139,7 @@ internal object DeviceInfo {
 
         // Nook Glowlight 3 et al.
         NOOK_V520 = (MANUFACTURER.contentEquals("barnesandnoble") || MANUFACTURER.contentEquals("freescale"))
-                && (MODEL.contentEquals("bnrv520") || MODEL.contentEquals("bnrv700") || MODEL.startsWith("ereader"))
+                && (MODEL.contentEquals("bnrv520") || MODEL.contentEquals("bnrv700") || MODEL.startsWith("ereader")) || MODEL.startsWith("bnrv510" ||
```
Comment (pazos) [inline app/src/org/koreader/launcher/DeviceInfo.kt]:
```suggestion
                && (MODEL.contentEquals("bnrv510") || (MODEL.contentEquals("bnrv520") || MODEL.contentEquals("bnrv700") || MODEL.startsWith("ereader"))
```

**Comment by hcors** [comment]
Thank you, noted.

**Comment by hcors** [comment]
> Cool, thanks for your work!

Honestly I don't know how I wasn't able to replicate that nomenclature successfully. I swear I have a brain, long day at work :)

**Pull request merged**